<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/login.png</normaloff>:/images/login.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/images/background/login_background.png);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>330</width>
     <height>220</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <widget class="QLabel" name="label_welcom">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>351</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/images/background/welcom_words.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="editHdbIp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRegister">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>250</y>
      <width>82</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>82</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid #cfcff1;		border-radius: 1px;	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #f6f7fa, stop: 1 #dadbde);	min-width: 80px;
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>注 册</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLogin">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>250</y>
      <width>82</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>82</width>
      <height>20</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid #cfcff1;		border-radius: 2px;	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #f6f7fa, stop: 1 #dadbde);	min-width: 80px;
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>登 录</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>82</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>82</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">   border: 1px solid #cfcff1;		border-radius: 2px;	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #f6f7fa, stop: 1 #dadbde);	min-width: 80px;
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>退 出</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>110</y>
      <width>138</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/images/background/透明底图1.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>学 员 用 户 名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editUserName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>150</y>
      <width>137</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/images/background/透明底图1.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string> 登  录  密  码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>190</y>
      <width>139</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/images/background/透明底图1.png);
font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string> 历史库服务器：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editUserName</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>buttonLogin</tabstop>
  <tabstop>buttonExit</tabstop>
  <tabstop>buttonRegister</tabstop>
  <tabstop>editHdbIp</tabstop>
 </tabstops>
 <resources>
  <include location="../SimuTrainee.qrc"/>
 </resources>
 <connections/>
</ui>
